app/scripts/filter/text.js

```diff
@@ -120,4 +120,9 @@ angular.module('common')
     return function(timestamp) {
       return new Date(timestamp).toUTCString();
     };
+  })
+  .filter('formatGeoshape', function() {
```

Comment:
filter name and function implementation do not match.
I would name the filter `urlEncodedJson` (or similar)

if it is not too expensive calling multiple filters on an object I would even make two filters out of it, one for `JSON.stringify` and one for `encodeURIComponent`